src/adapters/test-plugin/index.js

```diff
@@ -0,0 +1,11 @@
+/**
+ * index.js - Loads the Test adapter.
```

Comment:
I think the test-plugin-adapter should be renamed to `example-plugin-adapter` so that its purpose is clearer

Reply:
Sounds good.